.travis.yml

```diff
@@ -12,11 +12,12 @@ branches:
 
 services:
  - mongodb
+ - docker
 
 install:
  - npm install
 
 before_install:
+ - docker pull fiware/orion:latest
```

Comment:
Looking this in detail, .travis.yml file in the two other main IOTAs (UL and JSON) they aren't deploying Orion in before_install:

https://github.com/telefonicaid/iotagent-json/blob/master/.travis.yml#L21
https://github.com/telefonicaid/iotagent-ul/blob/master/.travis.yml#L21

So, why in the case of IOTA LWM2M is different?

Reply:
The reason is that tests for IOTA LWM2M are not really unit tests since the responses from the Orion CB are not mocked. Instead, an instance of the CB must be deployed and the tests are executed against it.
I guess that we should open a new issue in order to mock CB in the unit tests of IOTA LWM2M.

Reply:
Issue created at https://github.com/telefonicaid/lightweightm2m-iotagent/issues/106. NTC with regards this PR.